oneanddone/tests/functional/conftest.py

```diff
@@ -50,12 +50,24 @@ def new_user(persona_test_user):
 
 
 @pytest.fixture(scope='session')
-def base_url(base_url, live_server):
-    return base_url or live_server.url
+def base_url(base_url, request):
+    return base_url or request.getfuncargvalue("live_server").url
```

Comment:
Interesting, I was aware of this, but wasn't aware it invoked the fixture too! Awesome!